TrashMob/Controllers/PartnerUsersController.cs

```diff
@@ -11,26 +11,27 @@
     using System.Threading.Tasks;
```

Comment:
Before I refactor any other controller files, I want to make sure I did this one correctly. Am I on the right track or am I missing something?